src/emqx_auth_ldap.erl

```diff
@@ -133,3 +188,35 @@ do_resolve(HashType, Passhash, Password) ->
 
 description() -> "LDAP Authentication Plugin".
 
+compile_filters([{Key, Value}], []) ->
```

Comment:
I assume it just works, but would be great to see unit/functional tests for this later on.

Reply:
Agreed. It will come later, if the current implementation works.

Reply:
I added a config that exercises this logic.

Reply:
Since I don't think this filter declaration is a long-lasting solution, I'd rather not spend too much time writing tests.